lib/rewire/module.ex

```diff
@@ -143,6 +142,16 @@ defmodule Rewire.Module do
     )
   end
 
+  defp rewrite(use_ast = {:use, l1, [{:__aliases__, l2, macro_ast} | _args]}, acc) do
+    # We need to expand macros so we can rewire their code.
+    debug_log(acc, "expanding macro: #{inspect(macro_ast)}")
+    {Macro.expand(use_ast, __ENV__), acc}
+  end
+  defp rewrite(using_ast = {{:., _, [macro_ast, :__using__]}, _, _}, acc) do
+    # TODO
```

Comment:
I tried `{Macro.expand(using_ast, __ENV__), acc}` again, but that doesn't seem to work